### PR TITLE
Done & clean.

### PR DESCRIPTION
**Pull request by hidalgoss**
Please, review clean PR.

### CODE REVIEW

File: benchmark/benchmark_json/include/BmPods.h [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+                pods::JsonSerializer<pods::ResizableOutputBuffer> serializer(out);                  // Generate Buffer needed by next deserialization.
+                if(serializer.save(vecPodsInitialConditions) != pods::Error::NoError){
+                    std::cerr << "serialization error\n";
+                }
+            }
+
+            size_t retrievedVectorSize = 0;
+            for (auto _ : state) 
+            {   
+                state.PauseTiming();
```
Comment (pablogh-2000) [inline benchmark/benchmark_json/include/BmPods.h:72]: Remove these pause.

File: benchmark/benchmark_json/include/BmPods.h [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+                pods::JsonSerializer<pods::ResizableOutputBuffer> serializer(out);                  // Generate Buffer needed by next deserialization.
+                if(serializer.save(vecPodsInitialConditions) != pods::Error::NoError){
+                    std::cerr << "serialization error\n";
+                }
+            }
+            
+            size_t retrievedVectorSize = 0;
+            for (auto _ : state) 
+            {
+                state.PauseTiming();
```
Comment (pablogh-2000) [inline benchmark/benchmark_json/include/BmPods.h:170]: Remove pause

File: benchmark/benchmark_json/include/BmPods.h [hunk 3]
```diff
@@ -0,0 +1,566 @@
...
+                if (serializer.save(vecPodsInitialConditions) != pods::Error::NoError)
+                {
+                    std::cerr << "serialization error\n";
+                }
+            }
+
+            size_t retrievedVectorSize = 0;
+            for (auto _ : state) 
+            {
+                state.PauseTiming();
```
Comment (pablogh-2000) [inline benchmark/benchmark_json/include/BmPods.h:255]: remove pause

File: benchmark/benchmark_json/include/BmPods.h [hunk 4]
```diff
@@ -0,0 +1,566 @@
...
+                if (serializer.save(vecPodsInitialConditions) != pods::Error::NoError)
+                {
+                    std::cerr << "serialization error\n";
+                }
+            }
+
+            size_t retrievedVectorSize = 0;
+            for (auto _ : state) 
+            {   
+                state.PauseTiming();
```
Comment (pablogh-2000) [inline benchmark/benchmark_json/include/BmPods.h:339]: Remove pause

File: benchmark/benchmark_json/include/BmPods.h [hunk 5]
```diff
@@ -0,0 +1,566 @@
...
+                if (serializer.save(vecPodsInitialConditions) != pods::Error::NoError)
+                {
+                    std::cerr << "serialization error\n";
+                }
+            }
+
+            size_t retrievedVectorSize = 0;
+            for (auto _ : state) 
+            {
+                state.PauseTiming();
```
Comment (pablogh-2000) [inline benchmark/benchmark_json/include/BmPods.h:423]: Remove pause

File: benchmark/benchmark_json/include/BmPods.h [hunk 6]
```diff
@@ -0,0 +1,566 @@
...
+                if (serializer.save(vecPodsInitialConditions) != pods::Error::NoError)
+                {
+                    std::cerr << "serialization error\n";
+                }
+            }
+
+            size_t retrievedVectorSize = 0;
+            for (auto _ : state) 
+            {   
+                state.PauseTiming();
```
Comment (pablogh-2000) [inline benchmark/benchmark_json/include/BmPods.h:507]: Remove pause

**Comment by tomasz3dk** [comment]
Already merged with CI

**Pull request closed without merging**